package.json

```diff
@@ -29,110 +29,13 @@
-        "onCommand:vscode-docker.images.tag",
-        "onCommand:vscode-docker.images.copyFullTag",
-        "onCommand:vscode-docker.installDocker",
-        "onCommand:vscode-docker.networks.configureExplorer",
-        "onCommand:vscode-docker.networks.create",
-        "onCommand:vscode-docker.networks.inspect",
-        "onCommand:vscode-docker.networks.prune",
-        "onCommand:vscode-docker.networks.refresh",
-        "onCommand:vscode-docker.networks.remove",
-        "onCommand:vscode-docker.pruneSystem",
-        "onCommand:vscode-docker.registries.azure.buildImage",
-        "onCommand:vscode-docker.registries.azure.createRegistry",
-        "onCommand:vscode-docker.registries.azure.deleteRegistry",
-        "onCommand:vscode-docker.registries.azure.deleteRepository",
-        "onCommand:vscode-docker.registries.azure.openInPortal",
-        "onCommand:vscode-docker.registries.azure.runFileAsTask",
-        "onCommand:vscode-docker.registries.azure.runTask",
-        "onCommand:vscode-docker.registries.azure.untagImage",
-        "onCommand:vscode-docker.registries.azure.viewProperties",
-        "onCommand:vscode-docker.registries.azure.viewTaskLogs",
-        "onCommand:vscode-docker.registries.connectRegistry",
-        "onCommand:vscode-docker.registries.copyImageDigest",
-        "onCommand:vscode-docker.registries.copyRemoteFullTag",
-        "onCommand:vscode-docker.registries.deleteImage",
-        "onCommand:vscode-docker.registries.deployImageToAzure",
-        "onCommand:vscode-docker.registries.deployImageToAca",
-        "onCommand:vscode-docker.registries.disconnectRegistry",
-        "onCommand:vscode-docker.registries.dockerHub.openInBrowser",
-        "onCommand:vscode-docker.registries.help",
-        "onCommand:vscode-docker.registries.logInToDockerCli",
-        "onCommand:vscode-docker.registries.logOutOfDockerCli",
-        "onCommand:vscode-docker.registries.pullImage",
-        "onCommand:vscode-docker.registries.pullRepository",
-        "onCommand:vscode-docker.registries.reconnectRegistry",
-        "onCommand:vscode-docker.registries.refresh",
-        "onCommand:vscode-docker.volumes.configureExplorer",
-        "onCommand:vscode-docker.volumes.inspect",
-        "onCommand:vscode-docker.volumes.prune",
-        "onCommand:vscode-docker.volumes.refresh",
-        "onCommand:vscode-docker.volumes.remove",
-        "onCommand:vscode-docker.contexts.use",
-        "onCommand:vscode-docker.contexts.remove",
-        "onCommand:vscode-docker.contexts.inspect",
-        "onCommand:vscode-docker.contexts.configureExplorer",
-        "onCommand:vscode-docker.contexts.refresh",
-        "onCommand:vscode-docker.contexts.help",
         "onTaskType:docker-build",
         "onTaskType:docker-run",
         "onTaskType:docker-compose",
         "onDebugInitialConfigurations",
         "onDebugResolve:docker",
         "onFileSystem:docker",
-        "onLanguage:dockerfile",
```

Comment:
I verified that activation still occurs when a Dockerfile is opened. The activation event for compose is not removed, but the comma at the end of the line has to be removed so it shows up as an added line later.